drivers/hmis/lib/tasks/ce_define_workflows.rake

```diff
@@ -106,111 +105,322 @@ module CeWorkflowBuilder
+              { "code": 'No longer experiencing homelessness' },
+              { "code": 'Estimated vacancy no longer available' },
+              { "code": 'Enrolled, but declined HMIS data entry' },
+            ],
+            "mapping": { "custom_field_key": 'ac_workflow_v1_provider_denial_reason' },
+            "enable_when": [{ "question": 'move_forward', "operator": 'EQUAL', "answer_code": '0' }],
+          },
+        ],
+      },
+    )
+
+    create_step_form(
+      identifier: denial_review_form_identifier,
       definition: {
         "item": [
           {
-            "text": "Date",
-            "type": "DATE",
-            "link_id": "date",
+            "text": 'Date',
+            "type": 'DATE',
+            "link_id": 'date',
             "required": true,
-            "mapping": {"custom_field_key": "ce_generic_date"},
+            "mapping": { "custom_field_key": 'ce_generic_date' },
           },
           {
-            "text": "Notes",
-            "type": "TEXT",
-            "link_id": "notes",
+            "text": 'Notes',
+            "type": 'TEXT',
+            "link_id": 'notes',
             "required": false,
-            "mapping": {"custom_field_key": "ce_generic_notes"}
+            "mapping": { "custom_field_key": 'ce_generic_notes' },
           },
           {
-            "text": "Move forward?",
-            "type": "CHOICE",
-            "link_id": "move_forward",
+            "text": 'Decision',
+            "type": 'CHOICE',
+            "link_id": 'ac_workflow_v1_denial_review_decision',
             "required": true,
+            "component": 'RADIO_BUTTONS',
             "pick_list_options": [
               {
-                "code": "1",
-                "label": "Yes, move forward"
+                "code": '1',
+                "label": 'Approve Denial',
               },
               {
-                "code": "0",
-                "label": "No, decline referral"
-              }
+                "code": '0',
+                "label": 'Send Back',
+              },
             ],
-            "mapping": {"custom_field_key": "ce_generic_move_forward"}
+            "mapping": { "custom_field_key": 'ac_workflow_v1_denial_review_decision' },
           },
-        ]
-      }
+          {
+            "text": 'Reason for Sending Back',
+            "type": 'CHOICE',
+            "link_id": 'denial_reason',
+            "component": 'RADIO_BUTTONS',
+            "required": false,
+            "pick_list_options": [
+              { "code": 'HMIS user error' },
+              { "code": 'Client should be eligible' },
+            ],
+            "mapping": { "custom_field_key": 'ac_workflow_v1_denial_review_reason' },
+            "enable_when": [{ "question": 'denial_review_decision', "operator": 'EQUAL', "answer_code": '0' }],
+          },
+        ],
+      },
+    )
+    create_step_form(
+      identifier: confirm_success_task_form_identifier,
+      definition: ce_staff_shared_form,
```

Comment:
For the sake of making this into a sensible demo, maybe we don't want to use the shared form for this step? At this point in the referral, a target enrollment has just been created, so asking "Continue with Referral?" sounds odd to me.